www/templates/hostmaster-new-apphosting

```diff
@@ -0,0 +1,16 @@
+Hi Hostmaster,
+
+Could you please drop the following records to the DNS, if they exist:
+
+{hostname}.Berkeley.EDU. IN CNAME death.OCF.Berkeley.EDU.
+{hostname}.Berkeley.EDU. IN A 169.229.226.23
+{hostname}.Berkeley.EDU. IN AAAA 2607:f140:8801::1:23
```

Comment:
we shouldn't say "if they exist" -- that's super lazy and makes us look bad. it also makes it look like we don't understand how dns works, since we've listed both a cname and A records.

it takes five seconds to check the dns before emailing...

Reply:
All the records are listed so you can edit it to match whatever case it ends up being. I'll remove "if they exist," just seemed like a reasonable catchall.